README.md

```diff
@@ -103,6 +103,7 @@ Parameters:
 
 - `response_payload: string` - A string describing the payload type to return when fetching transaction metadata, e.g. "us-dl". See [our testing guide](https://docs.berbix.com/docs/testing) for possible options.
```

Comment:
While you are at it, can you mention that `response_payload` is a required field?  Maybe just add "(required)"?

Reply:
done!

Reply:
awesome, thanks!